src/app/app.module.ts

```diff
@@ -32,29 +32,37 @@ import {AppAuthService} from './app-auth.service';
 import {AppToolbarComponent} from './app-toolbar/app-toolbar.component';
 import {reducers} from './store/app.reducers';
 import { SessionService } from './session/session.service';
+import { LoginDialogComponent } from './login-dialog/login-dialog.component';
+import {FormBuilder} from '@angular/forms';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 
 @NgModule({
   declarations: [
     AppComponent,
     AppToolbarComponent,
     AppHomeComponent,
     AppWelcomeComponent,
-    LoginComponent,
+    LoginDialogComponent,
   ],
   imports: [
     HttpClientModule,
     BrowserModule,
     BrowserAnimationsModule,
     AppRoutingModule,
     FlexLayoutModule,
+    MatDialogModule,
     // SessionModule,
     AppMaterialModule,
     StoreModule.forRoot(reducers),
+    FormsModule,
```

Comment:
I'd add it after FlexLayoutModule to keep the angular modules the third parties and our own ones separated. Maybe a newline would be a good separator between them.  https://github.com/DD-DeCaF/caffeine/pull/10/files#diff-fc6b22833aeaf77f8440c4970ade705eR56 the